### DISCUSSION TITLE
Contribution Guidelines [Open for discussion]

### DISCUSSION THREAD

**Discussion by tbaden** [OpenMage/magento-lts#2106] [Ideas]
As a suggestion:
Creating a Guideline for good Code/Modules.
- Naming conventions
- module structure
- how to handle version numbers
- general rulesets
- basic rules for reviewing 
and so on...

as an example see this:
[oca guidelines](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst)
[migration guidelines](https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0)

**Comment by kkrieger85** [comment]
Maybe we could use https://github.com/magento/marketplace-eqp to ensure code quality

**Comment by kkrieger85** [comment]
@tbaden Maybe we should bring this discussion into https://github.com/OpenMage/organizational/issues
